tests/testsuite/cargo_information/invalid_path/stderr.log

```diff
@@ -0,0 +1,2 @@
+error: crates-io is replaced with remote registry dummy-registry;
```

Comment:
This is a bug. We should say this path doesn't have a workspace here.

Reply:
Oh, I guess it's not a real bug. Only happens when we have the replacement registry.

Reply:
Fixed in [`3cbf77f` (#83)](https://github.com/hi-rustin/cargo-information/pull/83/commits/3cbf77f20fbe2a2c7d180bcd5e1c3ca6500745a6)